examples/appliance_time_and_locale_configuration.py

```diff
@@ -27,38 +27,42 @@
     }
 }
 
+options = {
+    "locale": "en_US.UTF-8",
+    "timezone": "UTC",
+    "ntpServers": ["127.0.0.1"],
+}
+
+
 # Try load config from a file (if there is a config file)
 config = try_load_from_file(config)
-
 oneview_client = OneViewClient(config)
+time_and_locales = oneview_client.appliance_time_and_locale_configuration
 
 # Lists the appliance time and locale configuration
-time_and_locale = oneview_client.appliance_time_and_locale_configuration.get()
-print("\n## Got appliance time and locale configurations successfully!")
-pprint(time_and_locale)
+time_and_locale = time_and_locales.get_all()
+if time_and_locale:
+    print("\n## Got appliance time and locale configurations successfully!")
+    pprint(time_and_locale)
+else:
+    # Create a time and locale with the options provided
+    time_and_locale = time_and_locales.create(data=options)
+    print("\nCreated appliance time and locale configurations successfully!")
+    pprint(time_and_locale.data)
 
-# Backup original values
-bkp = {}
-bkp['ntpServers'] = time_and_locale['ntpServers']
-bkp['locale'] = time_and_locale['locale']
-
-# Update NTP servers and locale
+time_and_locale = time_and_locale.data
 # Set to use appliance local time and date server
 time_and_locale['ntpServers'] = ['127.0.0.1']
 # Set locale to Chinese (China) with charset UTF-8
 time_and_locale['locale'] = 'zh_CN.UTF-8'
 # Remove the date and time, we do not want to update it manually
 time_and_locale.pop('dateTime')
-time_and_locale = oneview_client.appliance_time_and_locale_configuration.update(time_and_locale)
+time_and_locale = time_and_locales.create(data=time_and_locale)
 print("\n## Updated appliance time and locale configurations successfully!")
```

Comment:
There is no update operation for this resource. So change the print statement as below.
```
print("\n## Changed the appliance time and locale configurations successfully!")
```